packages/@lwc/wire-service/src/property-trap.ts

```diff
@@ -42,20 +42,22 @@ function invokeConfigListeners(
 /**
  * Marks a reactive parameter as having changed.
  * @param cmp The component
- * @param reactiveParameter Reactive parameter that has changed
+ * @param reactiveParameters Reactive parameters that has changed
  * @param configContext The service context
  */
 export function updated(
     cmp: EventTarget,
-    reactiveParameter: ReactiveParameter,
+    reactiveParameters: Set<ReactiveParameter>,
     configContext: ConfigContext
 ) {
     if (!configContext.mutated) {
         configContext.mutated = new Set<ReactiveParameter>();
         // collect all prop changes via a microtask
         Promise.resolve().then(updatedFuture.bind(undefined, cmp, configContext));
     }
-    configContext.mutated.add(reactiveParameter);
+    reactiveParameters.forEach(reactiveParameter => {
+        configContext.mutated!.add(reactiveParameter);
```

Comment:
The `!` looks strange especially because there is an if branch above checking for the presence of the `mutated`  property in the `configContext`.

Reply:
is because of the function, with a regular for loop is fine (will take care once i update the pr)